library/include/modules/Units.h

```diff
@@ -99,34 +166,11 @@ DFHACK_EXPORT df::language_name *getVisibleName(df::unit *unit);
 DFHACK_EXPORT df::identity *getIdentity(df::unit *unit);
 DFHACK_EXPORT df::nemesis_record *getNemesis(df::unit *unit);
 
-DFHACK_EXPORT bool isHidingCurse(df::unit *unit);
 DFHACK_EXPORT int getPhysicalAttrValue(df::unit *unit, df::physical_attribute_type attr);
 DFHACK_EXPORT int getMentalAttrValue(df::unit *unit, df::mental_attribute_type attr);
 DFHACK_EXPORT bool casteFlagSet(int race, int caste, df::caste_raw_flags flag);
 
-DFHACK_EXPORT bool isCrazed(df::unit *unit);
-DFHACK_EXPORT bool isOpposedToLife(df::unit *unit);
-DFHACK_EXPORT bool hasExtravision(df::unit *unit);
-DFHACK_EXPORT bool isBloodsucker(df::unit *unit);
-DFHACK_EXPORT bool isMischievous(df::unit *unit);
-
-DFHACK_EXPORT df::unit_misc_trait *getMiscTrait(df::unit *unit, df::misc_trait_type type, bool create = false);
-
-DFHACK_EXPORT bool isDead(df::unit *unit);
-DFHACK_EXPORT bool isAlive(df::unit *unit);
-DFHACK_EXPORT bool isSane(df::unit *unit);
-DFHACK_EXPORT bool isCitizen(df::unit *unit);
-DFHACK_EXPORT bool isFortControlled(df::unit *unit);
-DFHACK_EXPORT bool isDwarf(df::unit *unit);
-DFHACK_EXPORT bool isWar(df::unit* unit);
-DFHACK_EXPORT bool isHunter(df::unit* unit);
-DFHACK_EXPORT bool isAvailableForAdoption(df::unit* unit);
-DFHACK_EXPORT bool isOwnCiv(df::unit* unit);
-DFHACK_EXPORT bool isOwnGroup(df::unit* unit);
-DFHACK_EXPORT bool isOwnRace(df::unit* unit);
-DFHACK_EXPORT bool isVisible(df::unit* unit);
-/// is unit hidden to the player? accounts for ambushing
-DFHACK_EXPORT bool isHidden(df::unit *unit);
+        DFHACK_EXPORT df::unit_misc_trait *getMiscTrait(df::unit *unit, df::misc_trait_type type, bool create = false);
```

Comment:
remove rogue indent

Reply:
counter: indent all of the declarations?

Reply:
either is good

Reply:
I guess I'll keep this in mind for the next one lol